cmd/internal/find/params/core.go

```diff
@@ -0,0 +1,8 @@
+// Package params represents `wash find`'s parameters. These are typically
+// set in `wash find`'s main function.
+package params
```

Comment:
Open to a better name for this, or another location. Main reason for the separate package is to avoid an import cycle (`meta` => `primary` => `meta`).

Reply:
None stand out to me. Params seems fine.